.idea/libraries/org_antlr_antlr4_4_5_1_1.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```

Comment:
This seems a bit too IDE-specific to be in the repository.
Although, I like the idea of using some package manager for java (maven?) to get the dependencies instead of the (rather ugly) javatools.jar...